sdk/messaging/azwebpubsub/health_client_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azwebpubsub_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
+	"github.com/Azure/azure-sdk-for-go/sdk/messaging/azwebpubsub"
+	"github.com/stretchr/testify/require"
+)
+
+func TestHealthClient_GetServiceStatus(t *testing.T) {
+	if recording.GetRecordMode() == recording.PlaybackMode || testing.Short() {
```

Comment:
This is causing the test to be skipped as the default mode is `PlaybackMode`. Also, why is there `testing.Short()`?

Comment applies elsewhere.